package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "internship",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.60",
+    "@types/react": "^18.2.34",
+    "@types/react-dom": "^18.2.14",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
```

Comment:
Why did you decide to keep "@testing-library/..." and "react-scripts" into "dependencies" instead of "devDependencies"?

Reply:
This was in the standsrt package.json created via cra, but i'll change it